project_types/tile_map_service/compare/project.py

```diff
@@ -74,3 +76,9 @@ def create_tasks(self, group: ProjectTaskGroup, raw_group: tile_grouping.RawGrou
                 tasks_count += 1
         bulk_mgr.done()
         return tasks_count
+
+    @typing.override
+    def get_project_specifics_for_firebase(self, project_ref):
+        class EmptyModel(BaseModel): ...
+
+        return EmptyModel()
```

Comment:
Maybe we should add ProjectSpecificFirebaseEmptyModel in the base (or in firebase models) and use that?
It will be easy to check which projects are using this.

Reply:
This will be very temporary, so we might not need to have a common empty class.

I have already replaced most of these usecases on another branch.

Reply:
Done!